src/main/java/com/blade/mvc/http/HttpRequest.java

```diff
@@ -51,7 +51,7 @@
 
     private Map<String, String>       headers    = null;
     private Map<String, Object>       attributes = null;
-    private Map<String, List<String>> parameters = null;
+    private Map<String, List<String>> parameters = new HashMap<>();
```

Comment:
this line of code has been initialized in the init method in order to make the performance better

Reply:
I see. What's your opinion about fixing the null pointer exception in MethodArgumentTest and RequestInvokerTest? By the way, `mvn test` actually fails but ci shows passing.

Reply:
I merged your PR, passed, thank you for your contribution.